homeassistant/components/victron_ble/strings.json

```
@@ -0,0 +1,298 @@
+{
+  "common": {
+    "low_voltage": "Low Voltage",
+    "high_voltage": "High Voltage",
+    "starter_voltage": "Starter Voltage",
+    "midpoint_voltage": "Midpoint Voltage"
```

We need sentence-casing on all strings!

```suggestion
    "low_voltage": "Low voltage",
    "high_voltage": "High voltage",
    "starter_voltage": "Starter voltage",
    "midpoint_voltage": "Midpoint voltage"
```

Same for all the new ones you just added.